### PR TITLE
feat: Bump kube-prometheus-stack to 44.2.3

### PR DESCRIPTION
**Pull request by Azhovan**
**What problem does this PR solve?**:
Back port to 2.5
chart changes:  https://github.com/mesosphere/charts/pull/1449 

**Which issue(s) does this PR fix?**:
https://d2iq.atlassian.net/browse/D2IQ-98716

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: licenses.d2iq.yaml [hunk 1]
```diff
@@ -340,24 +340,12 @@ resources:
       - url: https://github.com/kubernetes-retired/external-storage
         ref: nfs-provisioner-${image_tag}
         license_path: LICENSE
-  - container_image: quay.io/prometheus-operator/prometheus-operator:v0.62.0
-    sources:
-      - url: https://github.com/prometheus-operator/prometheus-operator
-        ref: ${image_tag}
-        license_path: LICENSE
-        notice_path: NOTICE
```
Comment (tillt) [inline licenses.d2iq.yaml:348]: This is weird - why are we losing those dependency images. I understand we likely are on a new version, but those new versions should appear here, no?

**Comment by cwyl02** [comment]
KPS `44.2.3` is not published for some reasons 😓

**Comment by Azhovan** [comment]
> mander-applications/manifest-

I locally ran `make publish` and now i can install it via 
```shell 
 helm install bb mesosphere-github-io-charts-staging/kube-prometheus-stack --version 44.2.3
```

**Review by cwyl02 — APPROVED** [review]

File: licenses.d2iq.yaml (hunk 1 above)
Comment (Azhovan) [reply licenses.d2iq.yaml]: https://github.com/mesosphere/kommander-applications/actions/runs/6147909730/job/16680418101

**Pull request merged**